src/applications/vaos/containers/VAFacilityPage.jsx

```diff
@@ -229,6 +232,7 @@ export class VAFacilityPage extends React.Component {
             continueLabel=""
             disabled={disableSubmitButton}
             pageChangeInProgress={loadingEligibility || pageChangeInProgress}
+            loadingText="Page change in progress"
```

Comment:
I think we need to do something like this here:

`loadingText={loadingEligibility ? "Checking eligibility requirements" : "Page change in progress"}`

Reply:
Done.